MeetupPlatformApi/Features/Authentication/RegisterNewUser/RegistrationResultDto.cs

```diff
@@ -8,7 +8,7 @@ public class RegistrationResultDto
     public UserInfoDto UserInfo { get; set; }
 
     [Required]
-    public string AccessToken { get; set; }
+    public TokenPairDto TokenPairInfo { get; set; }
```

Comment:
Consider renaming this property to just `TokenPair`, since we are not providing info about token pair, but the token pair itself.